BaseFilter/sections/specific.txt

```diff
@@ -841,6 +841,7 @@ techarmor.xyz##.my-ad-container
 !
 ! NOTE: Specific rules
 !
+havefunporn.com#%#//scriptlet('prevent-window-open')
```

Comment:
Please add the domain to this rule https://github.com/AdguardTeam/AdguardFilters/pull/220291#pullrequestreview-3587409862 like suggested.
Also block the leftover by adding the domain to the common rule `##.block-video > div.table`.